lib-utilities/logs/logging.go

```diff
@@ -116,10 +116,10 @@ func (l *Logging) auditLogEntry(ctx iris.Context, reqBody map[string]interface{}
 	reqStr := MaskRequestBody(reqBody)
 
 	// formatting logs in syslog format
-	if reqStr == "null" {
+	if reqStr == "" {
```

Comment:
This change may require changes in https://github.com/ODIM-Project/ODIM/blob/development/lib-utilities/logs/logging.go#L142